From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 17 Aug 2026 03:54:53 +0000
Subject: Translated using Weblate (Russian)

Currently translated at 100.0% (139 of 139 strings)

Translation: openkylin-huanghe/kom-service-support
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kom-service-support/ru/
---
 .../translations/kom-service-support_ru.ts         | 274 ++++++++++-----------
 1 file changed, 137 insertions(+), 137 deletions(-)

diff --git a/plugins/service-support/translations/kom-service-support_ru.ts b/plugins/service-support/translations/kom-service-support_ru.ts
index e517de3..faff829 100644
--- a/plugins/service-support/translations/kom-service-support_ru.ts
+++ b/plugins/service-support/translations/kom-service-support_ru.ts
@@ -6,117 +6,117 @@
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="35"/>
         <source>select detailed category</source>
-        <translation type="unfinished"></translation>
+        <translation>Пожалуйста, выберите детальную категорию</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="39"/>
         <source>System</source>
-        <translation type="unfinished"></translation>
+        <translation>Использование системы</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="41"/>
         <source>System activation</source>
-        <translation type="unfinished"></translation>
+        <translation>Активация системы</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="41"/>
         <source>System installation</source>
-        <translation type="unfinished"></translation>
+        <translation>Установка системы</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="41"/>
         <source>System crash</source>
-        <translation type="unfinished"></translation>
+        <translation>Сбой системы</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="42"/>
         <source>System performance</source>
-        <translation type="unfinished"></translation>
+        <translation>Производительность системы</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="42"/>
         <source>Control center</source>
-        <translation type="unfinished"></translation>
+        <translation>Панель управления</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="42"/>
         <source>System setting</source>
-        <translation type="unfinished"></translation>
+        <translation>Настройки системы</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="43"/>
         <source>System basis consulting</source>
-        <translation type="unfinished"></translation>
+        <translation>Базовые вопросы по системе</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="51"/>
         <source>Please describe in detail the problem you encountered, such as: unable to activate the system, can not find the relevant Settings, not clear system features, etc. </source>
-        <translation type="unfinished"></translation>
+        <translation>Здесь вы можете сообщить о проблемах, связанных с использованием системы, например: невозможно активировать систему, не удается найти соответствующие настройки, непонятна функциональность встроенных возможностей и т. д. Пожалуйста, максимально подробно опишите возникшую проблему. Вы также можете нажать кнопку ниже, чтобы загрузить изображения или файлы, чтобы мы могли быстрее решить проблему. </translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="64"/>
         <source>Please describe in detail the problems you encountered, such as: peripheral connection failure, sharing function Settings, peripheral adaptation, etc. </source>
-        <translation type="unfinished"></translation>
+        <translation>Здесь вы можете сообщить о проблемах, связанных с периферийными устройствами, например: невозможность подключения периферии, настройки функции общего доступа, совместимость периферийных устройств и т. д. Пожалуйста, максимально подробно опишите возникшую проблему. Вы также можете нажать кнопку ниже, чтобы загрузить изображения или файлы для более быстрого решения проблемы. </translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="75"/>
         <source>Please describe in detail the problems that you encounter, such as obtaining, installing, and uninstalling Kirin software errors. </source>
-        <translation type="unfinished"></translation>
+        <translation>Здесь вы можете сообщить о проблемах, связанных с программным обеспечением, например: получение, установка и удаление программного обеспечения Kylin и т. д. Пожалуйста, максимально подробно опишите возникшую проблему. Вы также можете нажать кнопку ниже, чтобы загрузить изображения или файлы, чтобы мы могли быстрее решить проблему. </translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="85"/>
         <source>Please describe your problem in detail, or you can also fill in your request or comment here.</source>
-        <translation type="unfinished"></translation>
+        <translation>Здесь вы можете сообщить о проблемах в других аспектах или предоставить нам ценные предложения.</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="149"/>
         <source>Attachment size exceeds limit!</source>
-        <translation type="unfinished"></translation>
+        <translation>Размер вложения превышает лимит!</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="57"/>
         <source>Peripheral</source>
-        <translation type="unfinished"></translation>
+        <translation>Консультация по периферийным устройствам</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="59"/>
         <source>Peripheral adaptation consulting</source>
-        <translation type="unfinished"></translation>
+        <translation>Консультация по совместимости периферийных устройств</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="59"/>
         <source>Peripheral driver acquisition</source>
-        <translation type="unfinished"></translation>
+        <translation>Получение драйверов внешних устройств</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="60"/>
         <source>Peripheral use and error reporting</source>
-        <translation type="unfinished"></translation>
+        <translation>Использование периферийных устройств и сообщения об ошибках</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="70"/>
         <source>Application</source>
-        <translation type="unfinished"></translation>
+        <translation>Прикладное программное обеспечение</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="72"/>
         <source>Software installation and uninstallation</source>
-        <translation type="unfinished"></translation>
+        <translation>Установка и удаление программного обеспечения</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="72"/>
         <source>Software use and error reporting</source>
-        <translation type="unfinished"></translation>
+        <translation>Использование программного обеспечения и ошибки</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="81"/>
         <source>Other</source>
-        <translation type="unfinished"></translation>
+        <translation>Прочее</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/feedbackmanager.cpp" line="83"/>
         <source>Opinions and suggestions</source>
-        <translation type="unfinished"></translation>
+        <translation>Предложения и замечания</translation>
     </message>
 </context>
 <context>
@@ -124,7 +124,7 @@
     <message>
         <location filename="../service-support-backend/src/feedbackmanagerlogic.cpp" line="61"/>
         <source>Failed to create temporary directory!</source>
-        <translation type="unfinished"></translation>
+        <translation>Не удалось создать временную директорию!</translation>
     </message>
 </context>
 <context>
@@ -132,12 +132,12 @@
     <message>
         <location filename="../UI/paginationwid.cpp" line="523"/>
         <source>Jump to</source>
-        <translation type="unfinished"></translation>
+        <translation>Перейти к</translation>
     </message>
     <message>
         <location filename="../UI/paginationwid.cpp" line="525"/>
         <source>Page</source>
-        <translation type="unfinished"></translation>
+        <translation>Страница</translation>
     </message>
 </context>
 <context>
@@ -145,22 +145,22 @@
     <message>
         <location filename="../UI/paginationwid.cpp" line="10"/>
         <source>total</source>
-        <translation type="unfinished"></translation>
+        <translation>Всего</translation>
     </message>
     <message>
         <location filename="../UI/paginationwid.cpp" line="11"/>
         <source>pages</source>
-        <translation type="unfinished"></translation>
+        <translation>Страница</translation>
     </message>
     <message>
         <location filename="../UI/paginationwid.cpp" line="40"/>
         <source>Jump to</source>
-        <translation type="unfinished"></translation>
+        <translation>Перейти к</translation>
     </message>
     <message>
         <location filename="../UI/paginationwid.cpp" line="43"/>
         <source>page</source>
-        <translation type="unfinished"></translation>
+        <translation>Страница</translation>
     </message>
 </context>
 <context>
@@ -168,47 +168,47 @@
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="191"/>
         <source>System log</source>
-        <translation type="unfinished"></translation>
+        <translation>Системный журнал</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="192"/>
         <source>Machine</source>
-        <translation type="unfinished"></translation>
+        <translation>Информация о всей машине</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="193"/>
         <source>Hardware</source>
-        <translation type="unfinished"></translation>
+        <translation>Параметры оборудования</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="194"/>
         <source>Drive</source>
-        <translation type="unfinished"></translation>
+        <translation>Информация о драйверах</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="195"/>
         <source>APP list</source>
-        <translation type="unfinished"></translation>
+        <translation>Список программного обеспечения</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="196"/>
         <source>Rules</source>
-        <translation type="unfinished"></translation>
+        <translation>Терминальная политика</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="197"/>
         <source>Network</source>
-        <translation type="unfinished"></translation>
+        <translation>Состояние сети</translation>
     </message>
     <message>
         <location filename="../service-support-backend/src/settings.cpp" line="198"/>
         <source>System</source>
-        <translation type="unfinished"></translation>
+        <translation>Состояние системы</translation>
     </message>
     <message>
         <location filename="../plugin.cpp" line="19"/>
         <source>ServiceSupport</source>
-        <translation type="unfinished"></translation>
+        <translation>Поддержка сервиса</translation>
     </message>
 </context>
 <context>
@@ -216,66 +216,66 @@
     <message>
         <location filename="../UI/uimainpage.cpp" line="145"/>
         <source>Feedback</source>
-        <translation type="unfinished"></translation>
+        <translation>Обратная связь по проблемам</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="149"/>
         <source>Self service</source>
-        <translation type="unfinished"></translation>
+        <translation>Самообслуживание</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="89"/>
         <location filename="../UI/uimainpage.cpp" line="166"/>
         <source>Application usage exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Аномальное использование приложения</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="91"/>
         <location filename="../UI/uimainpage.cpp" line="178"/>
         <source>Network connection exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Аномалия сетевого подключения</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="93"/>
         <location filename="../UI/uimainpage.cpp" line="190"/>
         <source>Hardware exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Аппаратные неполадки</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="95"/>
         <location filename="../UI/uimainpage.cpp" line="199"/>
         <source>System exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Системная ошибка</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="132"/>
         <source>Which of the following exceptions have you encountered?</source>
-        <translation type="unfinished"></translation>
+        <translation>Какой тип аномалии вы обнаружили?</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="137"/>
         <source>Enter the problem you encountered</source>
-        <translation type="unfinished"></translation>
+        <translation>Введите проблему, с которой вы столкнулись</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="146"/>
         <source>Submit problem opinions for feedback</source>
-        <translation type="unfinished"></translation>
+        <translation>Отправить вопрос или предложение для обратной связи</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="150"/>
         <source>Follow us for updates</source>
-        <translation type="unfinished"></translation>
+        <translation>Следите за нами, чтобы узнавать новости</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="153"/>
         <source>History</source>
-        <translation type="unfinished"></translation>
+        <translation>История отзывов</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="154"/>
         <source>View submitted feedback</source>
-        <translation type="unfinished"></translation>
+        <translation>Просмотр отправленных отзывов</translation>
     </message>
     <message>
         <location filename="../UI/uimainpage.cpp" line="157"/>
@@ -285,7 +285,7 @@
     <message>
         <location filename="../UI/uimainpage.cpp" line="221"/>
         <source>More automated services</source>
-        <translation type="unfinished"></translation>
+        <translation>Больше самообслуживания</translation>
     </message>
 </context>
 <context>
@@ -293,27 +293,27 @@
     <message>
         <location filename="../UI/uifaq.cpp" line="46"/>
         <source>Application usage exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Аномальное использование приложения</translation>
     </message>
     <message>
         <location filename="../UI/uifaq.cpp" line="47"/>
         <source>Network connection exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Аномалия сетевого подключения</translation>
     </message>
     <message>
         <location filename="../UI/uifaq.cpp" line="48"/>
         <source>Hardware exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Аппаратные ошибки</translation>
     </message>
     <message>
         <location filename="../UI/uifaq.cpp" line="49"/>
         <source>System exception</source>
-        <translation type="unfinished"></translation>
+        <translation>Аномалия системы</translation>
     </message>
     <message>
         <location filename="../UI/uifaq.cpp" line="59"/>
         <source>Enter the problem you encountered</source>
-        <translation type="unfinished"></translation>
+        <translation>Введите проблему, с которой вы столкнулись</translation>
     </message>
 </context>
 <context>
@@ -329,102 +329,102 @@
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="155"/>
         <source>Creation time</source>
-        <translation type="unfinished"></translation>
+        <translation>Время отзыва</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="155"/>
         <source>Type</source>
-        <translation type="unfinished"></translation>
+        <translation>Тип проблемы</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="71"/>
         <source>bydesign</source>
-        <translation type="unfinished"></translation>
+        <translation>Так задумано</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="73"/>
         <source>duplicate</source>
-        <translation type="unfinished"></translation>
+        <translation>Повторяющийся вопрос</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="75"/>
         <source>external</source>
-        <translation type="unfinished"></translation>
+        <translation>Внешние причины</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="77"/>
         <source>fixed</source>
-        <translation type="unfinished"></translation>
+        <translation>Решено</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="79"/>
         <source>notrepro</source>
-        <translation type="unfinished"></translation>
+        <translation>Невозможно воспроизвести</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="81"/>
         <source>postponed</source>
-        <translation type="unfinished"></translation>
+        <translation>Обработка отложена</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="83"/>
         <source>willnotfix</source>
-        <translation type="unfinished"></translation>
+        <translation>Не будет решено</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="99"/>
         <source>verify</source>
-        <translation type="unfinished"></translation>
+        <translation>Подтвердить решение</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="102"/>
         <source>Has the issue been resolved?</source>
-        <translation type="unfinished"></translation>
+        <translation>Пожалуйста, подтвердите, решена ли эта проблема?</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="105"/>
         <source>Once identified, the issue will be closed and no further action will be taken.</source>
-        <translation type="unfinished"></translation>
+        <translation>После подтверждения решения проблема будет закрыта, и технический специалист больше не будет заниматься ею.</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="107"/>
         <source>resolved</source>
-        <translation type="unfinished"></translation>
+        <translation>Решено</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="109"/>
         <source>cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Отмена</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="155"/>
         <source>Description</source>
-        <translation type="unfinished"></translation>
+        <translation>Описание проблемы</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="155"/>
         <source>Progress</source>
-        <translation type="unfinished"></translation>
+        <translation>Прогресс обработки</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="194"/>
         <source>No record</source>
-        <translation type="unfinished"></translation>
+        <translation>Нет записей обратной связи</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="195"/>
         <source>There is a network problem, please try again later</source>
-        <translation type="unfinished"></translation>
+        <translation>Не удалось подключиться к серверу, попробуйте позже</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="196"/>
         <source>Loading, please wait</source>
-        <translation type="unfinished"></translation>
+        <translation>Загрузка, подождите немного</translation>
     </message>
     <message>
         <location filename="../UI/uihistoryfeedback.cpp" line="209"/>
         <source>retry</source>
-        <translation type="unfinished"></translation>
+        <translation>Повторить попытку</translation>
     </message>
 </context>
 <context>
@@ -432,198 +432,198 @@
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="112"/>
         <source>Advanced</source>
-        <translation type="unfinished"></translation>
+        <translation>Расширенный режим</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="117"/>
         <source>Type</source>
-        <translation type="unfinished"></translation>
+        <translation>Тип проблемы</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="135"/>
         <source>Please describe the problem in detail and you can upload a photo or file by clicking the button below.</source>
-        <translation type="unfinished"></translation>
+        <translation>Подробно опишите возникшую у вас проблему. Вы также можете нажать кнопку ниже, чтобы загрузить изображения или файлы для быстрого решения проблемы.</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="137"/>
         <source>Remaining</source>
-        <translation type="unfinished"></translation>
+        <translation>Осталось</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="138"/>
         <source>character</source>
-        <translation type="unfinished"></translation>
+        <translation>символов</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="175"/>
         <source>Details</source>
-        <translation type="unfinished"></translation>
+        <translation>Описание проблемы</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="187"/>
         <source>ScreenShot</source>
-        <translation type="unfinished"></translation>
+        <translation>Скриншот</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="203"/>
         <source>Add file</source>
-        <translation type="unfinished"></translation>
+        <translation>Добавить файл</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="261"/>
         <source>The phone number cannot be empty</source>
-        <translation type="unfinished"></translation>
+        <translation>Номер мобильного телефона не может быть пустым</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="263"/>
         <source>The phone number format is incorrect</source>
-        <translation type="unfinished"></translation>
+        <translation>Неверный формат номера мобильного телефона</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="268"/>
         <source>Please enter your phone number</source>
-        <translation type="unfinished"></translation>
+        <translation>Пожалуйста, введите свой номер телефона</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="274"/>
         <source>appellation</source>
-        <translation type="unfinished"></translation>
+        <translation>Обращение</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="282"/>
         <source>Contact</source>
-        <translation type="unfinished"></translation>
+        <translation>Контактная информация</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="286"/>
         <source>Email required for anonymous feedback, not required for gitee feedback</source>
-        <translation type="unfinished"></translation>
+        <translation>Почта обязательна для анонимных отзывов, можно не заполнять при использовании gitee для обратной связи</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="294"/>
         <source>The mailbox format is incorrect</source>
-        <translation type="unfinished"></translation>
+        <translation>Неправильный формат электронной почты</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="301"/>
         <location filename="../UI/uiproblemfeedback.cpp" line="806"/>
         <source>Log in to gitee</source>
-        <translation type="unfinished"></translation>
+        <translation>Войти в gitee</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="318"/>
         <source>Mailbox</source>
-        <translation type="unfinished"></translation>
+        <translation>Электронная почта</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="327"/>
         <source>Agree to take mine </source>
-        <translation type="unfinished"></translation>
+        <translation>Согласен предоставить мои </translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="337"/>
         <source>System information</source>
-        <translation type="unfinished"></translation>
+        <translation>Системная информация</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="363"/>
         <source>Submit</source>
-        <translation type="unfinished"></translation>
+        <translation>Отправить</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="380"/>
         <source>Details type</source>
-        <translation type="unfinished"></translation>
+        <translation>Подробная классификация</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="396"/>
         <source>Time period</source>
-        <translation type="unfinished"></translation>
+        <translation>Период сбоя</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="431"/>
         <source>Information</source>
-        <translation type="unfinished"></translation>
+        <translation>Информация о системе</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="434"/>
         <source>lately</source>
-        <translation type="unfinished"></translation>
+        <translation>Недавние</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="435"/>
         <source>days</source>
-        <translation type="unfinished"></translation>
+        <translation>дней</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="447"/>
         <source>YES</source>
-        <translation type="unfinished"></translation>
+        <translation>Да</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="457"/>
         <source>NO</source>
-        <translation type="unfinished"></translation>
+        <translation>Нет</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="479"/>
         <source>Upload log</source>
-        <translation type="unfinished"></translation>
+        <translation>Добавить журнал</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="485"/>
         <source>Path</source>
-        <translation type="unfinished"></translation>
+        <translation>Выбрать путь</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="613"/>
         <source>No more than 5 files and total capacity not exceed 10MB</source>
-        <translation type="unfinished"></translation>
+        <translation>Максимум можно загрузить 5 вложений общим размером не более 10 МБ</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="500"/>
         <source>Export to</source>
-        <translation type="unfinished"></translation>
+        <translation>Экспортировать в</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="614"/>
         <source>Supported formats: </source>
-        <translation type="unfinished"></translation>
+        <translation>Поддерживаемые форматы: </translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="136"/>
         <source>Up to 500 characters</source>
-        <translation type="unfinished"></translation>
+        <translation>Максимум 500 символов</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="240"/>
         <source>Files</source>
-        <translation type="unfinished"></translation>
+        <translation>Загрузить вложение</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="694"/>
         <source>Repeat addition</source>
-        <translation type="unfinished"></translation>
+        <translation>Повторное добавление</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="710"/>
         <source>Attachment size out of limit</source>
-        <translation type="unfinished"></translation>
+        <translation>Размер вложения превышает лимит</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="825"/>
         <source>Log out of gitee</source>
-        <translation type="unfinished"></translation>
+        <translation>Выйти из gitee</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="826"/>
         <source>gitee has been associated</source>
-        <translation type="unfinished"></translation>
+        <translation>Связано с gitee</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedback.cpp" line="932"/>
         <source>Add attachment</source>
-        <translation type="unfinished"></translation>
+        <translation>Добавить вложение</translation>
     </message>
 </context>
 <context>
@@ -631,59 +631,59 @@
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="37"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Подтвердить</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="45"/>
         <source>Retry</source>
-        <translation type="unfinished"></translation>
+        <translation>Повторить попытку</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="97"/>
         <source>Cancel successfully</source>
-        <translation type="unfinished"></translation>
+        <translation>Отменено успешно</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="101"/>
         <source>System is abnormal, contact technical support</source>
-        <translation type="unfinished"></translation>
+        <translation>Обнаружена аномалия вашей системы, пожалуйста, свяжитесь напрямую со специалистом техподдержки</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="102"/>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="109"/>
         <source>Log and submission is packed, please go</source>
-        <translation type="unfinished"></translation>
+        <translation>Журналы и отправленная информация упакованы, пожалуйста, перейдите в</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="102"/>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="109"/>
         <source>acquire</source>
-        <translation type="unfinished"></translation>
+        <translation>Получить</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="107"/>
         <source>Submission failed</source>
-        <translation type="unfinished"></translation>
+        <translation>Не удалось отправить ваш отзыв</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="108"/>
         <source>Click &apos;Retry&apos; to upload again, or contact us directly.</source>
-        <translation type="unfinished"></translation>
+        <translation>Нажмите «Повторить», чтобы снова загрузить, или свяжитесь со службой техподдержки напрямую.</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="40"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>Отмена</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="93"/>
         <source>Submitted successfully</source>
-        <translation type="unfinished"></translation>
+        <translation>Ваш отзыв успешно отправлен</translation>
     </message>
     <message>
         <location filename="../UI/uiproblemfeedbackdialog.cpp" line="130"/>
         <source>Under submission...</source>
-        <translation type="unfinished"></translation>
+        <translation>Отправка...</translation>
     </message>
 </context>
 <context>
@@ -691,37 +691,37 @@
     <message>
         <location filename="../UI/uiselfservice.cpp" line="18"/>
         <source>Contact us</source>
-        <translation type="unfinished"></translation>
+        <translation>Связаться с нами</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="39"/>
         <source>Mail</source>
-        <translation type="unfinished"></translation>
+        <translation>Электронная почта</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="52"/>
         <source>Community</source>
-        <translation type="unfinished"></translation>
+        <translation>Сообщество</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="62"/>
         <source>website</source>
-        <translation type="unfinished"></translation>
+        <translation>Официальный сайт</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="69"/>
         <source> to get more services</source>
-        <translation type="unfinished"></translation>
+        <translation> , чтобы узнать больше о сервисах</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="86"/>
         <source>Official account openKylin</source>
-        <translation type="unfinished"></translation>
+        <translation>Официальный аккаунт openKylin</translation>
     </message>
     <message>
         <location filename="../UI/uiselfservice.cpp" line="61"/>
         <source>Jump to</source>
-        <translation type="unfinished"></translation>
+        <translation>Нажмите, чтобы перейти к</translation>
     </message>
 </context>
 <context>
@@ -729,17 +729,17 @@
     <message>
         <location filename="../UI/uiservicesupport.cpp" line="24"/>
         <source>Feedback</source>
-        <translation type="unfinished"></translation>
+        <translation>Обратная связь по проблемам</translation>
     </message>
     <message>
         <location filename="../UI/uiservicesupport.cpp" line="25"/>
         <source>Self service</source>
-        <translation type="unfinished"></translation>
+        <translation>Самообслуживание</translation>
     </message>
     <message>
         <location filename="../UI/uiservicesupport.cpp" line="26"/>
         <source>History</source>
-        <translation type="unfinished"></translation>
+        <translation>История отзывов</translation>
     </message>
 </context>
 </TS>
